packages/vaadin-grid/theme/lumo/vaadin-grid-styles.js

```diff
@@ -138,13 +142,7 @@ registerStyles(
 
     [part~='row'][dragover][dragover='on-top'] [part~='cell']::after {
       height: 100%;
-    }
-
-    [part~='row'][dragstart] {
-      /* Add bottom-space to the row so the drag number doesn't get clipped. Needed for IE/Edge */
-      border-bottom: 100px solid transparent;
```

Comment:
This removal probably also fixes https://github.com/vaadin/web-components/issues/2124

Reply:
Yeah, that crossed my mind as well, but I didn't see if it has any impact there. I’ll do that now.

Reply:
Hmm, actually, I can't reproduce the bug in start.vaadin.com anymore, using Safari 15. So I can't make sure if this PR actually fixes the issue. Can you test it?

Reply:
I tested on Safari 15 and 14 but couldn't reproduce

Reply:
The implementation of that part of start.vaadin.com has changed as well, so it’s hard to test anymore.